pkcs11int.h

```diff
@@ -220,6 +222,37 @@ void register_##tt() {
     ce_Pkcs11_##classname->unserialize = zend_class_unserialize_deny;           \
 }
 
+#else
```

Comment:
This one is ugly.... having to duplicate the whole definition...

Reply:
+1 very ugly to maintain. But I do not see any alternative based on the current design. So this proposal LGTM but please you should add a XXX TODO comment that either the PHP 7 code could be phased out or another design is required.

Reply:
TODO added in second commit.

Reply:
LGTM